cfme/common/provider_views.py

```diff
@@ -554,6 +573,24 @@ class ContainerProviderEditViewUpdated(ContainerProviderEditView):
     metrics_type = BootstrapSelect(id='metrics_selection')
     alerts_type = BootstrapSelect(id='alerts_selection')
 
+    @View.nested
+    class proxy(Tab, BeforeFillMixin):  # NOQA
```

Comment:
__OPTIONAL__: Instead of duplicating ```proxy``` and `advanced` (in `ContainerProviderAddViewUpdated` and `MiddlewareProviderAddView`) - why not creating some `ProviderAddViewSettings` add inherit it?

Reply:
Done